tapisproject_cms/src/taccsite_cms/settings_custom.py

```diff
@@ -81,9 +86,3 @@
 INCLUDES_CORE_PORTAL = False
 INCLUDES_PORTAL_NAV = False
 INCLUDES_SEARCH_BAR = False
-
-########################
-# TACC: GOOGLE ANALYTICS
-########################
-
-GOOGLE_ANALYTICS_PROPERTY_ID = "G-5EQ8Y25ZTM"
```

Comment:
**Why?** Cuz this is already in (and belongs in) [local CMS settings file on TACC/Core-Portal-Deployments](https://github.com/TACC/Core-Portal-Deployments/blob/e91aaf4/tapisproject/camino/prod.cms.settings_local.py#L5).